rskj-core/src/test/java/co/rsk/RskTestUtils.java

```diff
@@ -18,11 +18,14 @@ public static Keccak256 createHash(int nHash) {
         return new Keccak256(bytes);
     }
 
+    public static ECKey getEcKeyFromSeed(String seed) {
+        byte[] serializedSeed = HashUtil.keccak256(seed.getBytes(StandardCharsets.UTF_8));
```

Comment:
```suggestion
        byte[] seedHash = HashUtil.keccak256(seed.getBytes(StandardCharsets.UTF_8));
```

I think this `serializedSeed` name might be confusing since serializing is simply arranging the structure or object into another format to easily store or transmit it. This is actually the `keccak256` hash of the seed, so, I think it should have a name that shows what it really is.

Reply:
`seedHashSerialized` ?